ocpmodels/trainers/base_trainer.py

```diff
@@ -794,3 +838,174 @@ def save_results(self, predictions, results_file, keys):
 
             logging.info(f"Writing results to {full_path}")
             np.savez_compressed(full_path, **gather_results)
+
+
+def interpolate_embeddings(
+    checkpoint_atom_embeddings,
+    fitted_datasets=["OC20", "OC22"],
+    additional_fitted_elements=None,
+    smoothing=0,
+):
+    # checkpoint_atom_embedding is tensor of shape (N_elements, embedding_size)
+    # datasets is a list of strings that represent stanard OCP datasets that the checkpoint might
+    #       have been trained on. It is used to guess which elements should be trusted
+    # fitted_elements_list is an an additional list of elements that were fitted, perhaps in case the dataset
+    #       that the dataset that this was trained on was not OC20 or OC22
+
+    OC20_elements = [
```

Comment:
Let's pull this and OC22_elements out of the base trainer to avoid clutter. Instead add files into https://github.com/Open-Catalyst-Project/ocp/tree/main/ocpmodels/datasets/embeddings and then import them to be used here.